specs/consul/SOURCES/consul-client.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Consul Client
+Documentation=https://www.consul.io/docs
+Requires=network.target remote-fs.target
+After=network.target remote-fs.target
+ConditionPathExists=/var/lib/consul
+
+[Service]
+Type=simple
+PIDFile=/var/run/consul-client/consul-client.pid
```

Comment:
If I understand it right, you run server and client under the same user. So maybe is better to use paths like `/var/run/consul/client.pid` and `/var/run/consul/server.pid`?

Reply:
Yes, it looks better, but I am a bit confused how to pack `/var/run/consul` to `consul-client` and `consul-server` and not to get into troubles with dependencies if we want to install both packages on the system.

Reply:
Yeah, you're right. I totally forgot that it two different sub-packages.